### DISCUSSION TITLE
Infinite debug messages

### DISCUSSION THREAD

**Discussion by sarmong** [yshui/picom#957] [Q&A]
When I run picom with `picom --log-level=DEBUG --config=/dev/null`, I get hundreds of the following messages per second.

```
[ 12/04/2022 10:54:54.612 x_print_error DEBUG ] X error 143 XCB_RENDER_PICTURE request 139 minor 5 serial 534
[ 12/04/2022 10:54:54.628 ev_handle DEBUG ] event    Event 0 serial 0x00001b5b window 0000000000 ""
```
![image](https://user-images.githubusercontent.com/42828375/205481532-c8f59366-e790-4f22-bfc0-b1ee954cb552.png)

I am running Arch. I tried both `picom-git`(882025) and picom from main repo (listed as 10.1-1 and `picom --version` returns `vgit-98a5c`)

**Comment by ivanesmantovich** [comment]
Same problem
![image](https://user-images.githubusercontent.com/70603816/208310979-cfb721f3-ee03-42a2-8ef1-0016a37edd89.png)

**Comment by yshui** [comment]
Fixed now.

**Reply by sarmong** [reply]
Still the same for me on `vgit-b700a`
